BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingViewController.swift

```diff
@@ -53,8 +53,15 @@ class DigitalInvoiceOnboardingViewController: UIViewController {
         title = .ginibankLocalized(resource: DigitalInvoiceStrings.screenTitle)
         view.backgroundColor = GiniColor(light: UIColor.GiniBank.light2, dark: UIColor.GiniBank.dark2).uiColor()
         contentView.backgroundColor = .clear
-        
-        topImageView.image = topImage
+
+        if let adapter = configuration.returnAssistantOnboardingIllustrationAdapter {
```

Comment:
[Request] You also need to call `pageDidAppear()` and `pageDidDisappear()` on the adapter to allow implementations to start/stop animations.